share/completions/emerge.fish

```diff
@@ -4,7 +4,7 @@
 
 function __fish_emerge_print_installed_pkgs --description 'Prints completions for installed packages on the system from /var/db/pkg'
  if test -d /var/db/pkg
-   find /var/db/pkg/ -type d | cut -d'/' -f5-6 | sort | uniq | \
+   find /var/db/pkg/ -type d | cut -d'/' -f5-6 | sort -u | \
```

Comment:
In general, we require POSIX compatibility (we have to rely on _something_). I don't think we have any special cases for non-compliant-by-choice systems, though we may work around bugs. 

Though I'm quite sure we have some things that require more than that by accident.